### PR TITLE
Csvoutput

### PR DESCRIPTION
**Pull request by david-deboer**
This gives the option for csv output for the hookup listing.  It is preparatory to improving the presentation of the data for commissioning.

### CODE REVIEW

File: hera_mc/cm_utils.py [hunk 1]
```diff
@@ -297,6 +297,18 @@ def html_table(headers, table):
     return s
 
 
+def csv_table(headers, table):
```
Comment (bhazelton) [inline hera_mc/cm_utils.py]: this function needs a docsting. Also, please do not use single letter variable names.

File: hera_mc/cm_hookup.py [hunk 2]
```diff
@@ -634,8 +634,8 @@ def show_hookup(self, hookup_dict, cols_to_show='all', state='full', ports=False
         ports:  boolean to include ports or not
         revs:  boolean to include revisions letter or not
         state:  show the full hookups only, or all
-        file:  file to use, None goes to stdout
-        output_format:  set to html for the web-page version, or ascii
+        filename:  file name to use, None goes to stdout
+        output_format:  set to html for the web-page version, or table, or csv
```
Comment (bhazelton) [inline hera_mc/cm_hookup.py]: what does the `table` format mean for file output?

Comment (david-deboer) [reply hera_mc/cm_hookup.py]: It writes the formatted table to that file.

Comment (bhazelton) [reply hera_mc/cm_hookup.py]: so it's a human-readable text file? Something explaining that needs to be in the docstring.

**Pull request merged**